### PR TITLE
Sentryerrorlogging

### PR DESCRIPTION
**Pull request by luismolinuevo**
This pr uses sentry for error logging. I also added some missing try catches

How to use:

Step One(Import on top):

import * as Sentry from "@sentry/nextjs";

Step Two(Call sentry):

Sentry.captureException(error);

Example:
import * as Sentry from "@sentry/nextjs";

try {

} catch(error) {
      Sentry.captureException(error);
}

### CODE REVIEW

File: app/create-acc/page.js [hunk 1]
```diff
@@ -47,6 +48,7 @@ export default function CreateAccount() {
       // Redirect to profile page or any other page as needed
       router.push("/profile");
     } catch (error) {
+      Sentry.captureException(error);  //need to add password checks for size, and etc to make this defualt
```
Comment (madarez) [inline app/create-acc/page.js:51]: Can you check @jg6604 's PR for this?

File: sentry.client.config.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+// This file configures the initialization of Sentry on the client.
+// The config you add here will be used whenever a users loads a page in their browser.
+// https://docs.sentry.io/platforms/javascript/guides/nextjs/
+
+import * as Sentry from "@sentry/nextjs";
+
+Sentry.init({
+  dsn: "https://8e4204152fca1a516844d444ebefdd57@o4507024374890496.ingest.us.sentry.io/4507024376856576",
```
Comment (madarez) [inline sentry.client.config.js:8]: Out of curiosity, do these URLs need to be obfuscated?

File: app/global-error.jsx
Comment (madarez) [inline app/global-error.jsx:1]: Can this perhaps be under `/components` folder?

Comment (luismolinuevo) [reply app/global-error.jsx:1]: Yes it could be.

File: .gitignore [hunk 3]
```diff
@@ -34,3 +34,9 @@ yarn-error.log*
 # typescript
 *.tsbuildinfo
 next-env.d.ts
+
+# Sentry Config File
+.sentryclirc
+
+# Sentry Config File
+.sentryclirc
```
Comment (madarez) [inline .gitignore:42]:
```suggestion

# Sentry Config File
.sentryclirc
```

Comment (luismolinuevo) [reply .gitignore]: done

File: app/global-error.jsx
Comment (luismolinuevo) [reply app/global-error.jsx:1]: done

**Comment by luismolinuevo** [comment]
@madarez my merging is blocked

**Review by madarez — APPROVED** [review]

**Pull request merged**